### PR TITLE
Additional playback controls

### PR DESCRIPTION
**Pull request by FL550**
Adds Prev/next and volume control

### CODE REVIEW

File: src/spotify-card.ts [hunk 1]
```diff
@@ -293,21 +294,32 @@ export class SpotifyCard extends LitElement {
   }
 
   private onShuffleSelect(): void {
-    if (this._spotify_state?.state == 'playing') {
```
Comment (FL550) [inline src/spotify-card.ts:296]: Why have you done this check? It works even when pasued

Comment (fondberg) [reply src/spotify-card.ts:296]: Right. Whqt happens if nothing is playing or is it hidden then?

Comment (FL550) [reply src/spotify-card.ts:296]: Nothing happens. No, it's always visible, I don't like it when the controls shift through the UI

Comment (FL550) [reply src/spotify-card.ts:296]: Maybe we can hide the controls when no device is selected

Comment (fondberg) [reply src/spotify-card.ts:296]: You can choose. I dont have an opinion really

**Comment by FL550** [comment]
I pushed already a version, which hides the controls when no playback device is selected. Are you okay with the changes?

**Pull request merged**